File: 0001-two-sum/0001-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        for idx1, num1 in enumerate(nums):
            for idx2, num2 in enumerate(nums):
                if num1 + num2 == target and idx1 != idx2:
                    return [idx1, idx2]

File: 0001-two-sum/0001-two-sum.cpp
#include <iostream>

using namespace std;

class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        
        vector<int> res;
        bool found = false;
        
        for (int i=0; i < nums.size() ; ++i){
            for (int j =0 ; j < nums.size(); ++j){
                if( i != j && nums[i]+nums[j] == target ){
                    res.push_back(i);
                    res.push_back(j);
                    found = true;
                }
            }
            if (found) break;
        }

        return res;
    }
};